lib/db.js

```diff
@@ -384,6 +397,27 @@ module.exports = function (
     )
   }
 
+  DB.prototype.updateSessionTokenInBackground = function (token, userAgentString) {
+    log.trace({ op: 'DB.updateSessionTokenInBackground', uid: token && token.uid })
+
+    var self = this
+    var d = P.defer()
+
+    // Even though pool.post is async, we still defer calling it
+    // because userAgent blocks
+    setImmediate(function () {
```

Comment:
This seems a bit much. Is `ua.parse` *that* slow? Or am I missing something else?